pkg/network/cloud_network.go

```diff
@@ -0,0 +1,39 @@
+package network
+
+import (
+	"context"
+	"fmt"
+
+	configv1 "github.com/openshift/api/config/v1"
+	operv1 "github.com/openshift/api/operator/v1"
+	"github.com/openshift/cluster-network-operator/pkg/bootstrap"
+	types "k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func bootstrapCloud(conf *operv1.Network, kubeClient client.Client) (*bootstrap.CloudBootstrapResult, error) {
+	var platformType configv1.PlatformType
+	var region string
+
+	infraConfig := &configv1.Infrastructure{}
+	if err := kubeClient.Get(context.TODO(), types.NamespacedName{Name: "cluster"}, infraConfig); err != nil {
+		return nil, fmt.Errorf("failed to get infrastructure 'config': %v", err)
+	}
+
+	if infraConfig != nil {
+		if infraConfig.Status.PlatformStatus.Type != "" {
+			platformType = infraConfig.Status.PlatformStatus.Type
+		}
+		if platformType == configv1.AWSPlatformType {
+			region = infraConfig.Status.PlatformStatus.AWS.Region
+		} else if platformType == configv1.GCPPlatformType {
+			region = infraConfig.Status.PlatformStatus.GCP.Region
```

Comment:
No need to get Azure's region?

Reply:
There is no "region" on Azure, at least not in `infraConfig.Status.PlatformStatus.Azure.Region`, and as mentioned above: we don't need it on Azure.